.github/workflows/contracts.yml

```diff
@@ -262,51 +262,53 @@ jobs:
       - name: Check formatting
         run: yarn format
 
-  contracts-slither:
-    needs: contracts-detect-changes
-    if: |
-      github.event_name == 'push'
-        || needs.contracts-detect-changes.outputs.path-filter == 'true'
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: ./solidity
-    steps:
-      - uses: actions/checkout@v2
-
-      - uses: actions/setup-node@v2
-        with:
-          node-version: "14.x"
-          cache: "yarn"
-          cache-dependency-path: solidity/yarn.lock
-
-      - uses: actions/setup-python@v2
-        with:
-          python-version: 3.8.5
-
-      - name: Install Solidity
-        env:
-          SOLC_VERSION: 0.8.9 # according to solidity.version in hardhat.config.ts
-        run: |
-          pip3 install solc-select
-          solc-select install $SOLC_VERSION
-          solc-select use $SOLC_VERSION
-
-      - name: Install Slither
-        env:
-          SLITHER_VERSION: 0.8.2
-        run: pip3 install slither-analyzer==$SLITHER_VERSION
-
-      # We need this step because the `@keep-network/tbtc` which we update in
-      # next steps has a dependency to `@summa-tx/relay-sol@2.0.2` package, which
-      # downloads one of its sub-dependencies via unathenticated `git://`
-      # protocol. That protocol is no longer supported. Thanks to this step
-      # `https://` is used instead of `git://`.
-      - name: Configure git to don't use unauthenticated protocol
-        run: git config --global url."https://".insteadOf git://
-
-      - name: Install dependencies
-        run: yarn install
-
-      - name: Run Slither
-        run: slither --hardhat-artifacts-directory build .
+  # The check commented out because it was bringing more problems than value.
+  # It was raising a lot of false flags, which had to be individually supressed.
```

Comment:
I would rather say we are commenting it out until https://github.com/crytic/slither/issues/1106 is resolved and the fix is released. I think Slither brings *some* value but it's just not working at the moment. We need a fix, then we can reconsider.

Reply:
Ok, I've forced-pushed a change - updated the comment and commit message.

Reply:
Can we say the same in the commit message and in this comment (line 265/266). Right now they are conflicting with each other.

Reply:
Sorry, that was a mistake with me not committing the change and only force-pushing the amended commit message 🤦 ... Should be good now.